torch/distributed/benchmarks/benchmark_ddp_rpc.py

```diff
@@ -0,0 +1,350 @@
+import argparse
+import io
+import os
+import random
+import shlex
+import subprocess
+import time
+
+import numpy as np
+import torch
+import torch.distributed as dist
+import torch.distributed.autograd as dist_autograd
+import torch.distributed.rpc as rpc
+import torch.multiprocessing as mp
+import torch.optim as optim
+from torch.distributed.optim import DistributedOptimizer
+from torch.distributed.rpc import RRef, TensorPipeRpcBackendOptions
+from torch.distributed.rpc.backend_registry import BackendType
+from torch.nn.parallel import DistributedDataParallel as DDP
+
+
+# Config
+NUM_TRAINERS = 8
+NUM_PS = 8
+
+NUM_EMBEDDINGS = 100
+EMBEDDING_DIM = 16
```

Comment:
Maybe keep the default as 1000000 x 64 since this is a benchmark and not a test.